inversify.config.ts

```diff
@@ -0,0 +1 @@
+import 'reflect-metadata';
```

Comment:
is this needed?

Reply:
Good question. I did not figure out how DI framework works.

Reply:
This import is needed for the decorators to work. It allows metadata to be added to the objects

Reply:
![image](https://user-images.githubusercontent.com/54070318/73530605-bbd9cc80-4410-11ea-9de6-0f16df475aa1.png)